terraform/projects/app-deploy/main.tf

```
@@ -287,6 +287,11 @@ resource "aws_iam_role_policy_attachment" "allow_reads_from_artefact_bucket" {
   policy_arn = "${data.terraform_remote_state.artefact_bucket.read_artefact_bucket_policy_arn}"
 }
 
+resource "aws_iam_role_policy_attachment" "related_links_jenkins_iam_role_policy_attachment" {
```

nit: since Terraform resources are namespaced by type (i.e. to refer to this resource you'd write `aws_iam_role_policy_attachment.related_links_jenkins_iam_role_policy_attachment`) it's slightly more readable to omit the suffix, like this:

```suggestion
resource "aws_iam_role_policy_attachment" "related_links_jenkins" {
```

(definitely don't worry about changing any pre-existing ones though!)

---

Thanks, another good point. I was following past precedent and also my own past precedent, but totally makes sense what you suggest here.